Copy old docs (#7)
pgruetter/sdl-docs: docs/reference/metrics.md

---
id: metrics
title: Metrics
---

:::warning
This page is under review and currently not visible in the menu.
:::

## Metrics

> Metrics are gathered per Action and output-DataObject when running a DAG. They can be found in log statements and are written to the state file.

Reviewer: ```suggestion
While the DAG is running, metrics are gathered for each Action and output-DataObject. They can be found in log statements and are written to the state file.
```
